prometheus-to-sd/main.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"time"
+
+	"github.com/golang/glog"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	v3 "google.golang.org/api/monitoring/v3"
+
+	"k8s.io/contrib/prometheus-to-sd/config"
+	"k8s.io/contrib/prometheus-to-sd/translator"
+)
+
+var (
+	host        = flag.String("target-host", "localhost", "The monitored target's hostname.")
+	port        = flag.Uint("target-port", 80, "The monitored target's port.")
+	component   = flag.String("component", "uknown", "The monitored target's name.")
```

Comment:
Is metric name `container.googleapis.com/master/unknown/<some_name>` OK? Shouldn't it be required for user to specify?